ios/Podfile.lock

```diff
@@ -551,4 +551,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 217b8967e09f5059d7513e1729707b2ed0fb1a1b
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.9.3
```

Comment:
Intentional?

Reply:
At this point it's starting to feel intentional. 😂